spring.ts

```diff
@@ -41,9 +43,14 @@ export default function Spring(
       onUpdate = fn;
       fn(position);
     },
+    onComplete: (fn = v => {}) => {
+      onComplete = fn;
+      fn(position);
```

Comment:
Hmm also I’m not sure that this callback should be called immediately after subscription? I would find that a bit confusing but maybe I’m missing something?